oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -131,6 +131,22 @@ telemetry:
     port: ${SW_TELEMETRY_PROMETHEUS_PORT:1234}
 configuration:
   none:
+#  nacos:
+#    # Nacos Server Host
+#    serverAddr: 127.0.0.1
```

Comment:
Look like miss this in another yml?

Reply:
Added

Reply:
I mean there are two application.yml, another in assembly folder. Search it?

Reply:
Added